keras/utils/tracking.py

```diff
@@ -110,6 +113,7 @@ def add_to_store(self, store_name, value):
         self.stored_ids[store_name].add(id(value))
 
 
+@optree.register_pytree_node_class(namespace="keras")
```

Comment:
Why is this necessary?

Reply:
Because we need tree manipulations on `TrackedList`, `TrackedDict` and `TrackedSet` (as well as `ListWrapper`) and `optree` requires explicit registration

Please refer to https://github.com/metaopt/optree?tab=readme-ov-file#registering-a-container-like-custom-type-as-non-leaf-nodes

Reply:
Got it, that's pretty inconvenient but that makes sense.